.buildkite/pipeline.yml

```diff
@@ -29,3 +29,24 @@ steps:
     notify:
       - github_commit_status:
           context: "Unit tests"
+
+  - group: Prototype Build
+    steps:
+      - label: Package Prototype Build
+        key: prototype_build
+        command: .buildkite/commands/build_prototype.sh
+        plugins: [$CI_TOOLKIT]
+        artifact_paths:
+          - ".build/artifacts/*.ipa"
+          - ".build/artifacts/*.app.dSYM.zip"
+        notify:
+          - github_commit_status:
+              context: Package Prototype Build
+
+      - label: Distribute Prototype Build
+        depends_on: prototype_build
+        command: .buildkite/commands/distribute_prototype.sh
+        plugins: [$CI_TOOLKIT]
+        notify:
+          - github_commit_status:
+              context: Distribute Prototype Build
```

Comment:
Open to removing the `group` node. The new canvas makes the logical connection already clear:

<img width="809" alt="image" src="https://github.com/user-attachments/assets/e842fae4-01b0-4072-8189-23c754cc34fe" />

vs

<img width="873" alt="image" src="https://github.com/user-attachments/assets/0fee249b-798b-4581-bece-b17c0487c14c" />